src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
does it also work (if at all) for `distinct_hosts`?

if `Operand` is `distinct_host` the [https://www.nomadproject.io/docs/http/json-jobs.html](nomad json docs) says

```
If set, the scheduler will not co-locate any task groups on the same machine. This can be specified as a job constraint which applies the constraint to all task groups in the job, or as a task group constraint which scopes the effect to just that group. The constraint may not be specified at the task level.
```